packages/terra-aggregator/examples/components/DisclosureContent.jsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from 'terra-button';
+import ContentContainer from 'terra-content-container';
+import TextField from 'terra-form/lib/TextField';
+import AppDelegate from 'terra-app-delegate';
+import ActionHeader from 'terra-clinical-action-header';
+
+const propTypes = {
+  app: AppDelegate.propType,
+  name: PropTypes.string,
+};
+
+class DisclosureContent extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.checkLockState = this.checkLockState.bind(this);
+
+    this.state = {
+      text: undefined,
+    };
+  }
+
+  componentDidMount() {
+    if (this.props.app && this.props.app.registerLock) {
+      this.props.app.registerLock(this.checkLockState);
```

Comment:
I'm sorry I missed this, but how will any aggregator items (content) register locks? I see now how this works for any content disclosed, but nothing for anything in the main content:
https://github.com/cerner/terra-framework/blob/master/packages/terra-disclosure-manager/src/DisclosureManager.jsx#L257